bin/generators/credentials/list.js

```diff
@@ -6,33 +6,45 @@ module.exports = class extends eg.Generator {
 
     this.configureCommand({
       command: 'list [options]',
-      description: 'List all credentials for Consumer (User or App)',
+      description: 'List all active credentials for Consumer (User or App)',
```

Comment:
why active ? you can list inactive as well. previous description seems to be better